README.md

```diff
@@ -12,39 +12,27 @@ It also optionally supports subscription/unsubscription through a SaaS product o
 
 ### Prerequisites
 
-First, ensure you have the [latest AWS CLI installed](http://docs.aws.amazon.com/cli/latest/userguide/installing.html) (version >= 1.11.37) as well as [Node.js](https://nodejs.org/en/download/) 4+. Then, clone this repo into a local directory
+First, ensure you have the [latest version of the SAM CLI installed](https://docs.aws.amazon.com/lambda/latest/dg/sam-cli-requirements.html). Clone this repo into a local directory.
```

Comment:
I think we noted this in the bug bash, but mention something about not needing Docker & maybe a link to how to configure aws cli profiles.

Reply:
Yeah... None of the bug bash feedback is folded into the docs.